src/types/publication_types/ticket.types.ts

```diff
@@ -25,7 +25,7 @@ export interface TicketCollection {
 
 export type TicketType = 'DoiRequest' | 'GeneralSupportCase' | 'PublishingRequest';
 export type TicketStatus = 'New' | 'Pending' | 'Closed' | 'Completed' | 'NotApplicable';
-export const ticketStatusValues: TicketStatus[] = ['New', 'Pending', 'Closed', 'Completed', 'NotApplicable'];
+export const ticketStatusValues: TicketStatus[] = ['New', 'Pending', 'Closed', 'Completed'];
```

Comment:
Denne kunne sikkert ha blitt endret til `ticketStatusFilterValues`, men tror det kan skape ekstra jobb for @joachimjorgensen om det endres nå.